<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCCUI</class>
 <widget class="QMainWindow" name="DCCUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocs"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuNodegraph">
    <property name="title">
     <string>Nodegraph</string>
    </property>
    <widget class="QMenu" name="menuAdd_Nodes">
     <property name="title">
      <string>Add Nodes...</string>
     </property>
     <widget class="QMenu" name="menuTesting">
      <property name="title">
       <string>Testing Node Subcat</string>
      </property>
      <addaction name="actionTesting_Add_Node"/>
     </widget>
     <addaction name="menuTesting"/>
    </widget>
    <addaction name="menuAdd_Nodes"/>
    <addaction name="actionExtract_Node"/>
    <addaction name="actionIgnore_Node"/>
    <addaction name="actionEdit_Node"/>
    <addaction name="actionView_Node"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuNew_Floating_Window">
     <property name="title">
      <string>New Floating Window</string>
     </property>
     <addaction name="actionTest_Windows"/>
    </widget>
    <widget class="QMenu" name="menuSaved_Layouts_2">
     <property name="title">
      <string>Saved Layouts</string>
     </property>
     <addaction name="actionTest_Saved_Layouts"/>
    </widget>
    <addaction name="menuNew_Floating_Window"/>
    <addaction name="actionSave_Layout_2"/>
    <addaction name="menuSaved_Layouts_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuNodegraph"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionTesting">
   <property name="text">
    <string>Testing</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F1</string>
   </property>
  </action>
  <action name="actionDocs">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionIgnore_Node">
   <property name="text">
    <string>Ignore Node</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionExtract_Node">
   <property name="text">
    <string>Extract Node</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+X</string>
   </property>
  </action>
  <action name="actionEdit_Node">
   <property name="text">
    <string>Edit Node</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionView_Node">
   <property name="text">
    <string>View Node</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionSave_Layout">
   <property name="text">
    <string>Save Layout...</string>
   </property>
  </action>
  <action name="actionTesting_2">
   <property name="text">
    <string>Saved Layout Testing</string>
   </property>
  </action>
  <action name="actionTesting_Add_Node">
   <property name="text">
    <string>Testing Add Node</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionTest_Windows">
   <property name="text">
    <string>Test Windows</string>
   </property>
  </action>
  <action name="actionSave_Layout_2">
   <property name="text">
    <string>Save Layout</string>
   </property>
  </action>
  <action name="actionTest_Saved_Layouts">
   <property name="text">
    <string>Test Saved Layouts</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
